scrnaseq.Rmd

```diff
@@ -16,6 +16,87 @@ output:
 bibliography: "references.bib"
 ---
 
+```{css, echo=FALSE}
```

Comment:
Can we export this to `R/functions_css.R` or alike?

Reply:
I have exported into a `style.css` which is on the same level as the `scrnaseq.rmd`.

Reply:
`css` directory?

Reply:
Done, yes